### PR TITLE
Add support for adjusting the size of data objects

### PR DESCRIPTION
**Pull request by korydraughn**
Running test suite.

### CODE REVIEW

File: test/irods_tests.bats [hunk 1]
```diff
@@ -273,3 +273,45 @@ teardown() {
...
+    # Increase the size of the file.
+    local NEW_DATA_SIZE=256
+    truncate -s${NEW_DATA_SIZE} ${FILENAME}
+    run stat -c%s ${FILENAME}
+    #echo "# status = $status" >&3
+    #echo "# output = $output" >&3
+    [ "$status" -eq 0 ]
+    [ "$output" = "${NEW_DATA_SIZE}" ]
+
+    # Show the original contents is still there.
```
Comment (trel) [inline test/irods_tests.bats:309]: contents is -> contents are
or
contents is -> content is

Comment (korydraughn) [reply test/irods_tests.bats:309]: `content is` sounds more correct, but I'll do a little googling to be sure.

**Comment by korydraughn** [comment]
All tests passed.

File: test/irods_tests.bats (hunk 1 above)
Comment (korydraughn) [reply test/irods_tests.bats]: Done.

**Comment by trel** [comment]
nice.

please # them if you're ready.

**Comment by korydraughn** [comment]
#'d

**Pull request merged**